src/main/java/ch/njol/skript/expressions/ExprVectorLength.java

```diff
@@ -70,7 +71,7 @@ public void change(Event event, @Nullable Object[] delta, ChangeMode mode) {
 				//$FALL-THROUGH$
 			case ADD:
 				for (Vector vector : vectors) {
-					if ((deltaLength < 0 && vector.lengthSquared() < deltaLength * deltaLength) || vector.isZero()) {
+					if ((deltaLength < 0 && vector.lengthSquared() < deltaLength * deltaLength) || VectorMath.isZero(vector)) {
```

Comment:
It may be worthwhile to swap the order of these conditions, so that isZero is done first.
Short circuiting the multiplication would probably be a small improvement. It's really like the smallest detail possible though lol.

Reply:
Sure, why not